test/api.player.test.js

```diff
@@ -734,6 +734,67 @@ describe("Player API Tests", function() {
 		});
 	});
 
+	describe("Give Gift test", function() {
+
+		var sentPlayerId = window.mock.env.playerId;
+		var receivedPlayerId = window.mock.env.playerId2;
+
+		beforeAll(function(done) {
+			done();
+		});
+
+		it("Test give badge", function(done) {
+			Playbasis.engineApi.rule("fbcomment", sentPlayerId, {post_custom_params: {type: "badge"}})
+				.then((result) => {
```

Comment:
Could `.then` be indented with the same level at `Playbasis...`, because it's promise-chain call, it will totally look better and cleaner that way.

Like

```
Playbasis.engineApi.rule("fbcomment", sentPlayerId, {post_custom_params: {type: "badge"}})
.then((result) => {

```

Also for other occurences of `.then` in this commit too.

All in all, it looks great! Just above then we're good to go.